src/05-invoicing/invoices.apib

```diff
@@ -161,6 +161,22 @@ Get details for a single invoice.
             + created_at: `2016-02-04T16:44:33+00:00` (string)
             + updated_at: `2016-02-05T16:44:33+00:00` (string)
 
+### invoices.download [POST /invoices.download]
+
++ Request (application/json;charset=utf-8)
+
+    + Attributes (object)
+        + id: `d885e5d5-bacb-4607-bde9-abc4a04a901b` (string, required)
+        + format: `pdf` (enum[string], required)
+            + Members
+                + pdf
```

Comment:
Do I understand that right, that we actually only offer `pdf` as format but leave it open to have more supported formats in the future?

Reply:
Yep, this is to ensure we can add formats without breaking backwards compatibility.